nomos-blend/core/src/lib.rs

```diff
@@ -4,14 +4,7 @@ pub mod message_blend;
 pub mod persistent_transmission;
 
 pub enum BlendOutgoingMessage {
```

Comment:
This will make my work of integrating the new scheduler into the service much easier!